### PR TITLE
document some more Release Manager assumptions

### LINKED ISSUE #340: release instructions: link to Apache RAT
RAT is mentioned [here](https://github.com/apache/openwhisk-release/blame/master/docs/release_instructions.md#L145) but not linked to or any information on how to install and use it.

I believe this is the source: https://creadur.apache.org/rat/

And usage instructions should be something like:

```
cd stagingArea/artifacts/
rat openwhisk-*-sources.tar.gz
```

### LINKED ISSUE #341: release instructions: link to PGP key setup for Apache
It should link to https://infra.apache.org/new-committers-guide.html#set-up-security-and-pgp-keys

### LINKED ISSUE #343: document that only PMC members can do all release steps
These steps are only possible by PMC members:

1. add pgp key to KEYS (one time setup only)
2. upload to dist
3. publishing to npm (openwhisk-bot credentials are in private PMC subversion repo)

### PR DESCRIPTION
**Pull request by dgrove-oss**
Fixes #340 
Fixes #341 
Fixes #343

### CODE REVIEW

File: docs/release_instructions.md [hunk 1]
```diff
@@ -19,6 +19,24 @@
...
+since you have done it), first read the
+[Release Manager Setup](release_manager_setup.md) instructions.
+We have scripts that automate many of the tasks a Release Manager must
+perform, but these scripts assume you have followed the instructions
+in the Release Manager Setup before you start executing them.
+
+## Non-PMC Release Managers
+
+Actually publishing a release is an official act of the OpenWhisk
+Project Management Committee (PMC).  If you an OpenWhisk Committer,
```
Comment (rabbah) [inline docs/release_instructions.md:34]:
```suggestion
Project Management Committee (PMC).  If you are an OpenWhisk Committer,
```

File: docs/release_instructions.md [hunk 2]
```diff
@@ -19,6 +19,24 @@
...
+but not an OpenWhisk PMC member, you will need to pair-up with a PMC member to
+accomplish some of the steps.  In brief, any step that adds or removes
+an artifact from a public distributions channel must be executed by
+an OpenWhisk  PMC member.
```
Comment (rabbah) [inline docs/release_instructions.md:38]:
```suggestion
an OpenWhisk PMC member.
```

File: docs/release_instructions.md [hunk 3]
```diff
@@ -142,8 +160,13 @@ script will run rcverify.sh against your local artifacts.
 ./local_verify.sh ../release-configs/<MY_RELEASE_CONFIG>.json
 ```
 
-TODO:  We should also run Apache Rat as part of local_verify.sh; for now it is
-a recommended best practice to run Apache Rat by hand on each of your .tar.gz files.
+TODO:  We should also run Apache Rat (https://creadur.apache.org/rat/)
+as part of local_verify.sh; for now it is a recommended best practice
```
Comment (rabbah) [inline docs/release_instructions.md:164]:
```suggestion
as part of `local_verify.sh`; for now it is a recommended best practice
```

File: docs/release_instructions.md [hunk 4]
```diff
@@ -250,14 +274,17 @@ information into the release description.
 
 ### Dockerhub updates
 
+**Only a PMC member can push images to the openwhisk organization on dockerhub**
```
Comment (rabbah) [inline docs/release_instructions.md:277]:
```suggestion
**Only a PMC member can push images to the OpenWhisk organization on Docker Hub**
```

Comment (rabbah) [reply docs/release_instructions.md:277]: heh I see `dockerhub` is throughout... ignore this nit :)

**Comment by alexkli** [comment]
Thanks @dgrove-oss  👍

**Pull request merged**